website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -139,23 +139,23 @@ This POST endpoint requires a JSON object with the following properties as a req
+| `data.attributes.is-destroy`                       | bool                 | false                                              | Specifies whether this plan is a destroy plan that will destroy all provisioned resources. Mutually exclusive with `refresh-only`.                                                                                                                                                                                                                                                                                            |
+| `data.attributes.message`                          | string               | "Queued manually via the Terraform Enterprise API" | Specifies the message associated with this run.                                                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.refresh`                          | bool                 | true                                               | Specifies whether or not to refresh the state before a plan.                                                                                                                                                                                                                                                                                                                                                                  |
+| `data.attributes.refresh-only`                     | bool                 | false                                              | Whether this run should refresh the state without modifying any resources. Mutually exclusive with `is-destroy`.                                                                                                                                                                                                                                                                                                              |
+| `data.attributes.replace-addrs`                    | array\[string]       |                                                    | Specifies an optional list of resource addresses to be passed to the `-replace` flag.                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I'm going to leave this as is, because the way it currently is represented is clear to me and more concise:
![Screenshot 2024-01-08 at 4 43 51 PM](https://github.com/hashicorp/terraform-docs-common/assets/32436232/d2dc4c39-f36a-4dda-8e95-a18f17eaf456)